pyproject.toml

```diff
@@ -57,9 +57,19 @@ interfaces-vision = [
     "pillow>=6.2.1,<11.0"
 ]
 
+interfaces-ts = [
+    "pandas>=1.4.3,<2",
+]
+
+interfaces-ts-pyspark = [
+    "pandas>=1.4.3,<2",
+    "pyspark>=3.3,<3.5",
+    "pyarrow"
```

Comment:
Shoot, I think we need to put a range on this one

Reply:
Changed to "pyarrow>=8.0.0,<15"